neo/NeoSystem.cs

```diff
@@ -44,6 +44,10 @@ public void Dispose()
         {
             RpcServer?.Dispose();
             EnsureStoped(LocalNode);
+            foreach (var p in Plugin.Plugins)
+            {
+                p.Dispose();
+            }
```

Comment:
Some plugins may access `RpcServer`. So maybe we need to dispose the plugins first.